skipruntime-ts/adapters/postgres/src/index.ts

```
@@ -230,17 +230,19 @@ FOR EACH ROW EXECUTE FUNCTION %I();`,
   }
 
   unsubscribe(instance: string): void {
-    this.client
-      .query(format("DROP FUNCTION IF EXISTS %I CASCADE;", instance))
-      .then(
-        () => this.open_instances.delete(instance),
-        (e: unknown) => {
-          console.error(
-            "Error unsubscribing from resource instance: " + instance,
-          );
-          throw e;
-        },
-      );
+    if (this.open_instances.has(instance))
```

Do we want to throw a specific error in case the user attempts to `unsubscribe()` from an instance not in `open_instance`, as it would likely indicate a bug in user code (vs this package)?

---

At least for the time being, it doesn't necessarily indicate a bug in user code -- on the native side, `closeService_` calls `unsubscribe` on each resource instance as well as `shutdown` on the whole external service; I don't want to mess around with that or its ordering without consulting Daniel.

So this works around the semantics there, making the operations idempotent and also doing the "right" thing for isolated calls to unsubscribe and shutdown.

There could be cases where a user double-unsubscribes or unsubscribes an instance that never actually existed, but I'm not sure whether it's worth trying to catch that.